### DISCUSSION TITLE
[Feature Request] Integration with TaskbarX

### DISCUSSION THREAD

**Discussion by CrazyWolf13** [valinet/ExplorerPatcher#2354] [Ideas]
Hi
Due to to WIn11 changes TaskbarX by Chris Andriesson broke more or less completely, however using Explorer Patcher with Win10 Taskbar it's possible to still get somewhat of a nice Taskbar.

Is there any way to get default Win11 to work with some modification with ExplorerPatcher, without completely using win10 taskbar, because long term windows will probably remove win10 taskbar in win12.

**Comment by pyrates999** [comment]
In one build of the insider build of windows 11, it did have the windows 10 taskbar removed.  There was an earlier filed issue that asked about this and it linked to an article indicating that.
